smartmontools/smartd.service.in

```diff
@@ -3,11 +3,42 @@ Description=Self Monitoring and Reporting Technology (SMART) Daemon
 Documentation=man:smartd(8) man:smartd.conf(5)
 
 [Service]
-Type=notify
+AmbientCapabilities=CAP_SYS_RAWIO
+CapabilityBoundingSet=CAP_SYS_RAWIO
+DeviceAllow=/dev/null rw
+DeviceAllow=block-sd r
```

Comment:
This should conflict with many non-sd devices, e.g. usb, scsi, raid, etc. Not something i would like to add by default.

Reply:
I could add all remaining block devices:
```
DeviceAllow=block-blkext r
DeviceAllow=block-device-mapper r
DeviceAllow=block-md r
DeviceAllow=block-mdp r
DeviceAllow=block-sr r
```
I think USB and SCSI are `block-sd`.